webpack.config.js

```diff
@@ -69,7 +69,8 @@ module.exports = function (env, argv) {
         jsDir,
         "src/recommendations/Recommendations.tsx"
       ),
-      missingMBData: path.resolve(jsDir, "src/missing-mb-data/MissingMBData.tsx"),
+      missingMBData: path.resolve(jsDir, "src/MissingMBData.tsx"),
```

Comment:
I really don't know about this change -- feels wrong and unrelated, but the merge conflict brought it to light.

Reply:
In regards to https://github.com/metabrainz/listenbrainz-server/pull/1861, we have moved the frontend codebase to neat folders and hence this change appears.